modules/checklist/assets/js/editor.js

```diff
@@ -3,3 +3,15 @@ import Component from './component';
 
 $e.components.register( new Component() );
 editorV2();
+
+window.addEventListener( 'DOMContentLoaded', maybeHideToggleIcon );
+
+function maybeHideToggleIcon() {
```

Comment:
@ronenelementor Maybe we should conditionally run `editorV2()` based on the setting value, instead of always registering it and maybe hiding it. WDYT?

Reply:
I thought about it, but then if the user turns the settings switch on, we won't be able to show it as far as I understand, WDYT?

Reply:
You are correct! A page refresh is probably not the best UX.